region/maridia/outer/Main Street.json

```
@@ -3042,6 +3042,50 @@
       "resetsObstacles": ["R-Mode"],
       "flashSuitChecked": true
     },
+    {
+      "link": [3, 3],
+      "name": "R-Mode Spark Interrupt",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        "Gravity",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 60}}
+              ]},
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 1}]},
+                {"partialRefill": {"type": "ReserveEnergy", "limit": 40}}
+              ]}
+            ]}
+          ]}
+        ]},
+        {"or": [
+          {"ammo": {"type": "Super", "count": 1}},
+          "canBePatient"
+        ]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 24, "gentleUpTiles": 3, "gentleDownTiles": 3, "openEnd": 1}},
+          {"and": [
+            {"doorUnlockedAtNode": 3},
+            {"canShineCharge": {"usedTiles": 25, "gentleUpTiles": 3, "gentleDownTiles": 3, "openEnd": 1}}
```

no open ends because you dont want to fall off
```suggestion
          {"canShineCharge": {"usedTiles": 24, "gentleUpTiles": 3, "gentleDownTiles": 3, "openEnd": 0}},
          {"and": [
            {"doorUnlockedAtNode": 3},
            {"canShineCharge": {"usedTiles": 25, "gentleUpTiles": 3, "gentleDownTiles": 3, "openEnd": 0}}
```

---

It's because you can fall off for the in-room spark .... but yeah, I guess you could run left to right, so this can be kept how you had it.